.jenkins/pytorch/build.sh

```diff
@@ -30,6 +30,7 @@ cmake --version
 pip install -r requirements.txt || true
 
 if [[ "$BUILD_ENVIRONMENT" == *rocm* ]]; then
+  # This is necessary in order to cross compile (or else we'll have missing GPU device).
```

Comment:
I don't think this comment should have been duplicated here?

Reply:
probably happened in a merge. do you want me to update for this PR or just fix for a future one?